tensorflow/lite/micro/tools/gen_micro_mutable_op_resolver/generate_micro_mutable_op_resolver_from_model.py

```diff
@@ -23,7 +23,11 @@
 from absl import flags
 from mako import template
 
-from tflite_micro.tensorflow.lite.tools import visualize as visualize
+# TODO(#2564): Resolve import issue
+try:
+  from tflite_micro.tensorflow.lite.tools import visualize as visualize
+except ImportError:
+  from tensorflow.lite.tools import visualize as visualize
```

Comment:
I think I prefer to land your original change that just has:

```
from tensorflow.lite.tools import visualize
```

I just need to fix some internal stuff to land it.

Reply:
Ok, let's go for that. Done